src/main/java/edu/nitt/delta/bustracker/controller/api/DriverController.java

```diff
@@ -0,0 +1,41 @@
+package edu.nitt.delta.bustracker.controller.api;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import edu.nitt.delta.bustracker.model.Driver;
+import edu.nitt.delta.bustracker.service.DriverService;
+
+@RestController
+@RequestMapping("/driver")
+public class DriverController {
+
+    @Autowired 
+    private DriverService driverService;
+
+    @GetMapping
+    public List<Driver> getAllDriver() {
```

Comment:
For each route, enclose the code in try-catch blocks and send a 500 response in case of an exception